src/features/place-bid/PlaceBidButton/PlaceBidButton.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react';
+
+import { usePlaceBidData } from '../usePlaceBidData';
+
+import { PlaceBidModal } from '..';
+
+type PlaceBidButtonProps = {
+	domainId: string;
+	isRoot?: boolean;
+};
+
+export const PlaceBidButton: FC<PlaceBidButtonProps> = ({
+	domainId,
+	isRoot,
+}) => {
+	const { tokenBalance } = usePlaceBidData(domainId);
```

Comment:
Added here so the query has been fetched prior to the modal opening. If we call the query when the modal opens, the token balance for a split second will be undefined causing the content to jolt/flicker as there is a condition that checks if the user has a valid token balance